lang/build.gradle

```diff
@@ -113,9 +112,8 @@ runKtlintCheckOverMainSourceSet.dependsOn generatePigDomains
 
 ktlint {
     filter {
-        // excluding multiple files from ktlint was somewhat of a challenge using other approaches..
-        // We use the "workaround" described here: https://github.com/JLLeitschuh/ktlint-gradle/issues/222
-        exclude({it.file.absolutePath.contains("/generated-src/")})
+        // Excludes any code in the build directory, which is assumed to be generated.
+        exclude { projectDir.toURI().relativize(it.file.toURI()).path.startsWith("build") }
```

Comment:
Is this the best way? Doesn't gradle have `project.buildDir` which we could use?

Reply:
It seems weird that a linter would ever lint the build directory

Reply:
The ignore for the generated sources might actually just be copy-paste from the PIG package, and is likely unnecessary in the PartiQL package. Ktlint should ignore the build directory, but [this same configuration was added to the pig-tests package](https://github.com/partiql/partiql-ir-generator/pull/128/files#diff-825d89dbeee13ab092df4a6b3762752aec1d25ed867a797c85a22e4bcd75e214) because that generated source in _not_ in build/

Perhaps we can get away with removing this config altogether.

Reply:
update: the linter runs for everything in the main sourceSet, and pig sources are explicitly added hence why we need to ignore